### ISSUE TITLE
Keyboard navigation of the dataviewer is missing some stuff

### ISSUE THREAD

**Issue by rchiodo** [microsoft/vscode-jupyter#3195]
- Can't sort on column (at least space or enter didn't do anything)
- Seems to tab to the filter row even when hidden
